data/Arts2023Longitudinal.yaml

```diff
@@ -0,0 +1,3857 @@
+# yaml-language-server: $schema=.schema.yaml
+title: Longitudinal and quantitative fecal shedding dynamics of SARS-CoV-2, pepper
+  mild mottle virus, and crAssphage
+doi: 10.1128/msphere.00132-23
+description: >
+  The authors present longitudinal, quantitative fecal shedding data for SARS-CoV-2
+  RNA, pepper mild mottle virus (PMMoV) RNA, and crAss-like phage (crAssphage) DNA
+  from 48 COVID-19 patients. Abundances were quantified using (RT)-ddPCR assays targeting
+  the N and ORF1a genes. The data were obtained from supplementary material.
+analytes:
+  stool_SARSCoV2_N:
+    description: >
+      Concentration of RNA of the N gene quantified using (RT)-ddPCR in stool samples.
+      The concentration was quantified in gene copies per dry weight of stool. The
+      limit of blank (LOB), determined as the upper 95% confidence limit of the negative
+      extraction control, ranged from 11.2 to 1,550 gc/mg-dry weight.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 1550
+    specimen: stool
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_SARSCoV2_ORF1a:
+    description: >
+      Concentration of RNA of the ORF1a gene quantified using (RT)-ddPCR in stool
+      samples. The concentration was quantified in gene copies per dry weight of
+      stool. The limit of blank (LOB), determined as the upper 95% confidence limit
+      of the negative extraction control, ranged from 11.2 to 1,550 gc/mg-dry
+      weight.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 1550
+    specimen: stool
+    biomarker: SARS-CoV-2
+    gene_target: ORF1a
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_PMMoV:
+    description: >
+      Concentration of PMMoV RNA was measured by ddPCR on the same day as the SARS-CoV-2.
+      The concentration was quantified in gene copies per dry weight of stool.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 400
+    specimen: stool
+    biomarker: PMMoV
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_crAssphage:
+    description: >
+      Concentration of crAssphage DNA was measured using qPCR. The reported number
+      is either the measured concentration of crAssphage or the Limit of Quantification
+      (LOQ) if the concentration wasn't quantifiable.
```

Comment:
@tillahoffmann @WenfeiXiao , I think the negative results were all replaced with LODs (varied by sample).  In Liu2024Longitudinal.yaml, I use "negative" for all the negative samples in the `value`. Should we include `LOD` for each sample and use "negative" as the `value`? An example can be:
```
participants:
   -attributes:
    age: 44
    sex: female
  measurements:
    -analyte: stool_SARSCoV2_N
     time: 7
     value: negative
     LOD: 100
```

Reply:
Good question. It looks like the smallest reported value for `crAss_conc` is 2.114056386 but even values as high as 128.5881388 appear when `crAss_det` is `False`. Do you think it's reasonable for there to be a 50x difference between levels of quantification across different samples?

Reply:
Yes, this is possible as the sample process (e.g., concentration) and lab methods could introduce large variations in the LOD. And I have seen this in other datasets.

Reply:
Sounds good. This also records positive but not quantifiable results.

Reply:
Perfect, sounds like we have a plan. What are your thoughts, @WenfeiXiao?

Reply:
Sounds great! I'll make limit_of_quantification an option for each measurement with a negative limit of detection.